gdx/src/com/badlogic/gdx/graphics/g2d/GlyphLayout.java

```diff
@@ -122,6 +125,7 @@ public void setText (BitmapFont font, CharSequence str, int start, int end, Colo
 			if (start == end) { // End of text.
 				if (runStart == end) break; // No run to process, we're done.
 				runEnd = end; // Process the final run.
+				isLastRun = true;
```

Comment:
`isLastRun` seems identical to `runEnd == end`.

Reply:
Yes, it just is much easier to identify `isLastRun` than  `runEnd = end`. We can remove that again if you want.